pkgs/applications/networking/instant-messengers/pantalaimon/default.nix

```
@@ -51,6 +51,12 @@ buildPythonApplication rec {
   # darwin has difficulty communicating with server, fails some integration tests
   doCheck = !stdenv.isDarwin;
 
+  postPatch = ''
+    set -x
+    sed -i 's/"matrix-nio\[e2e\] <= 0.14"/"matrix-nio[e2e] >= 0.14, < 0.15"/' setup.py
+    cat ./setup.py
+  '';
```

looks like some debugging commands
```suggestion
  postPatch = ''
    sed -i 's/"matrix-nio\[e2e\] <= 0.14"/"matrix-nio[e2e] >= 0.14, < 0.15"/' setup.py
  '';
```